lib/Test/Pod/CoverageChange.pm

```diff
@@ -0,0 +1,188 @@
+=head1 DESCRIPTION
+
+Prints the percentage of POD coverage in **TODO** test format for currently existing packages.
+Prints B<not ok> (with respective message) if our latest changes increased/decreased numbers of naked functions in currently existing packages.
+Prints B<not ok> if currently existing packages have 100% coverage unless you remove the package from B<%naked_packages> variable.
+Prints a proper message for the newly added packages.
+
+prints B<ok> for the files that have no POD syntax error.
+prints B<not ok- There is no POD in the file> if the file has no POD at all. I put this into a TODO test so CircleCI's tests will pass.
+prints B<not ok- The number of errors in the POD structure> if the file has any error. It causes CircleCI's tests to fail.
+
+=cut
+use strict;
+use warnings;
+
+use Test::More;
+use Pod::Checker;
+use Pod::Coverage;
+use File::Find::Rule;
+use Test::Pod::Coverage;
+use Module::Path 'module_path';
+
+our $VERSION='0.01';
+
+use constant {
+  POD_SYNTAX_IS_OK => 0,
+  FILE_HAS_NO_POD => -1,
+};
+
+our @ISA = qw(Exporter);
+our @EXPORT_OK = qw(check);
+
+=pod
+
+=head2 check
+
+Todo: Update this pod
+
+Check all modules under the given directory against POD coverage and POD syntax
+
+=over 4
+
+=item * C<directory> An arrayref of directories to check pod against
+
+=item * C<naked_packages> A hashref that contains some packages which are allowed to have naked subs.
+
+=back
+
+=cut
+
+sub check {
+    my $path = shift;
+    my $naked_packages = shift // {};
+    my $ignored_packages = shift // [];
+
+    $path = [$path] unless ref $path eq 'ARRAY';
+    $ignored_packages = [$ignored_packages] unless ref $ignored_packages eq 'ARRAY';
+
+    check_pod_coverage($path, $naked_packages, $ignored_packages);
+    check_pod_syntax($path, $ignored_packages);
+
+    Test::Pod::CoverageChange->export_to_level(1, @_);
+}
+
+=head2 check_pod_coverage
+
+Todo:: complete this pod
+
+=cut
+
+sub check_pod_coverage {
+    my $directories = shift;
+    my $allowed_naked_packages = shift;
+    my $ignored_packages = shift;
+
+    check_existing_naked_packages($allowed_naked_packages, $ignored_packages) if keys %$allowed_naked_packages;
+
+    # Check for newly added packages PODs
+    my @ignored_packages = (keys %$allowed_naked_packages, @$ignored_packages);
+    foreach my $package (Test::Pod::Coverage::all_modules(@$directories)) {
+        next if @ignored_packages && (grep(/^$package$/, @ignored_packages));
+        pod_coverage_ok($package, {private => []});
+    }
+}
+
+=head2 check_pod_syntax
+
+Todo:: complete this pod
+
+=cut
+
+sub check_pod_syntax {
+    my $directories = shift;
+    my $ignored_packages = shift;
+
+    my @ignored_packages_full_path = ();
+    for (@$ignored_packages) {
+        my $file_path = module_path($_);
+        push(@ignored_packages_full_path, $file_path) if defined $file_path;
+    }
+
+    my @files_path = File::Find::Rule->file()
+                                     ->name( '*.p[m|l]' )
```

Comment:
generally alignment should match the level of indentation (1 for the sub, another for the incomplete statement), otherwise you end up with arbitrary spacing based just on function/package names